web/webpack.config.js

```diff
@@ -8,7 +9,20 @@ module.exports = {
     filename : "bootstrap.js",
   },
   mode : "development",
-  plugins : [ new CopyWebpackPlugin([ 'index.html' ]) ],
+  plugins :
+    [
+      new CopyWebpackPlugin({ patterns : [ 'index.html' ], }),
+      new WasmPackPlugin(
+        {
+          crateDirectory: path.resolve(__dirname, '.'),
+          args: "--log-level warn",
+
+          // the mode `development` makes `wasm-pack` build in `debug` mode.
+          // the mode `production` makes `wasm-pack` build in `release` mode.
+          forceMode: "development",
```

Comment:
I would rather want to control this from the command line with a flag. Is that possible? Could you add a comment about it?

Reply:
Good idea, yes I can do it.

Reply:
Done! Unfortunately the webpack parameter passing convention forces a stray `--` (like, `npm start -- --env=release`, `npm build -- --env=release`). That's because webpack can't receive parameters without `--dashes`, and npm needs an empty `--` to separate them from npm's own parameters.

The alternative would be to create new commands `npm start:release` and `npm build:release` (duplicate commands on the npm side instead of adding options to webpack). But then it doesn't work if you call webpack directly.

Reply:
Actually I think `npm run start:release` and `npm run build:release` is better, the other one was too verbose.